### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Updating and testing two Hook Guides to single pages. Merged in: bd-okta-438764-Event-Hook-guide-to-one-guide and bd-okta-438763-common-hook-set-up-steps-one-guide; testing re-directs. Both branches approved from devdoc reviewers

- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-438764](https://oktainc.atlassian.net/browse/OKTA-438764)
* [OKTA-436763](https://oktainc.atlassian.net/browse/OKTA-438763)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/main/index.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+2. From the `Add Package` drop-down box, search for the `express-basic-auth` and `body-parser` packages.
+3. Click each package to add to your project.
+
+The Inline Hook guides use [HTTP Basic Authentication](/books/api-security/authn/api-authentication-options/#http-basic-authentication) to authenticate the Okta Inline Hook API calls received by your Glitch external service. In your Okta org, you must encode the Glitch project username and password credentials in Base64 and add them as the **Authentication secret** when you activate the Inline Hook. Ensure that you add the scheme `Basic ` (including a space) as a prefix to the **Authentication secret** value.
+
+For example, the credential pair used in the Inline Hook examples is `admin:supersecret`, which encoded in Base64 is `YWRtaW46c3VwZXJzZWNyZXQ=`. Adding the scheme to this value, creates the Inline Hook **Authentication secret** value: `Basic YWRtaW46c3VwZXJzZWNyZXQ=`.
+
+After including the `npm` packages, add the following code snippet in your project.
+
+<StackSelector snippet="auth"/>
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/main/index.md:68]:
```suggestion
<StackSelector snippet="auth" noSelector />
```

File: packages/@okta/vuepress-site/docs/guides/event-hook-implementation/main/index.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+<StackSelector snippet="verification"/>
+
+## Parse the Event Hook request
+
+When a user is deactivated in the Okta org, your external service receives the Event Hook request from Okta, and must parse the Event Object to determine the user name or other data required by your external service.
+
+In this example, after parsing the Event Hook request, the code simply displays the deactivated user to the console, and then replies to Okta with an empty, successful response (200).
+
+<StackSelector snippet="parse-request"/>
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/event-hook-implementation/main/index.md:61]:
```suggestion
<StackSelector snippet="parse-request" noSelector />
```
Don't need a second selector on the same page.

File: packages/@okta/vuepress-site/docs/guides/event-hook-implementation/main/index.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+## Examine the Event Hook object
+
+The JSON body includes the properties accessed in this example, namely `target` and `displayName`. To see this or other Event Objects, call your Okta org with the [System Log API](/docs/reference/api/system-log), using the specific event type as a [filter parameter](/docs/reference/api/system-log/#filtering-results). For example:
+
+```JavaScript
+https://${yourOktaDomain}/api/v1/logs?filter=eventType eq "user.lifecycle.deactivated"
+```
+
+<StackSelector snippet="event-object"/>
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/event-hook-implementation/main/index.md:71]:
```suggestion
<StackSelector snippet="event-object" noSelector />
```

**Review by vanngo-okta — APPROVED** [review]
Added suggestions to remove subsequent stack selectors on a page. Otherwise, LGTM! 🚀

**Comment by brianduffield-okta** [comment]
Added Van's `noSelector` option suggestion and built locally. Looks great, Van. Thank you.

**Pull request merged**